test/performance/odp_l2fwd.c

```diff
@@ -864,6 +874,157 @@ static int run_worker_sched_mode_vector(void *arg)
 	return 0;
 }
 
+/*
+ * Packet IO worker thread using scheduled queues and event vector mode.
+ *
+ * arg  thread arguments of type 'thread_args_t *'
+ */
+static int run_worker_sched_mode_event_vector(void *arg)
```

Comment:
This is largely identical to run_worker_check_mode_vector(). I wonder the functions could be combined.